internal/reconciler/shoot/reconciler_test.go

```diff
@@ -61,6 +61,9 @@ var _ = Describe("Reconciler", func() {
 			Client: fakeClient,
 			Config: configv1alpha1.ShootControllerConfig{
 				SyncPeriod: &metav1.Duration{Duration: time.Hour},
+				OIDCConfig: &configv1alpha1.OIDCConfig{
+					Audiences: []string{configv1alpha1.DefaultOIDCClientID},
```

Comment:
In tests we usually hardcode values so we see a failing tests if the constant changes
```suggestion
					Audiences: []string{"garden"},
```